src/cmdstan-guide/mcmc_config.qmd

```diff
@@ -311,16 +486,24 @@ specifies the location of the auxiliary output file which contains sampler infor
 and the gradients on the unconstrained scale and log probabilities for all parameters in the model.
 By default, no auxiliary output file is produced.
 
-## Multiple chains in one executable {#sampler-num-chains}
+## Running multiple chains {#multi-chain-sampling}
+
+A Markov chain generates samples from the target distribution only after it has converged to equilibrium.
```

Comment:
converged to equilibrium -> approximate converged to equilibrium -OR- approximately converged to the stationary distribution

The issue is that we never really converge to the stationary distribution if we do not start at a draw from the stationary distribution.